pages/[id].js

```diff
@@ -0,0 +1,24 @@
+import ProjectDetails from "@/components/ProjectDetails";
+import { initialProjects } from "@/db/data";
+import { useRouter } from "next/router";
+import Link from "next/link";
+
+export default function ProjectDetailPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const currentProject = initialProjects.find(
+    (initialProject) => initialProject.id.toString() === id
```

Comment:
You should think about using strings for the IDs in the data.js. Because if you are planning on using a MongoDB database, the ObjectID that are automatically generated per object in the database are going to be strings anyway. 
Maybe it will be less work to change the data.js instead of converting the id here. 

But thats optional, just as a thought to consider :)

Reply:
good point. did that. thank you.